test/WebJobs.Script.Tests.Integration/Management/InstanceManagerTests.cs

```diff
@@ -355,7 +356,7 @@ public async Task ValidateContext_InvalidZipUrl_WebsiteUseZip_ReturnsError()
         {
             var environmentSettings = new Dictionary<string, string>()
             {
-                { EnvironmentSettingNames.AzureWebsiteZipDeployment, "http://invalid.com/invalid/dne" }
+                { EnvironmentSettingNames.AzureWebsiteZipDeployment, "http://invalid.test/invalid/dne" }
```

Comment:
It is best practice to use `.test` TLD for fake endpoints, as it is guaranteed this endpoint will not exist.